openlibrary/components/ObservationForm/components/ValueCard.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <div class="value-card">
+    <CardHeader
```

Comment:
Future PR: We should probably dissolve these components, `CardHeader` and `CardBody`, and move all their logic up to here. They seem not super necessary.